scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -54,7 +54,7 @@
 
               {% if variant.category == "snv" and variant.chromosome in ["MT","M"] %}
                 <td>
-                  {% if sample.alt_frequency and sample.alt_frequency != -1 %}
+                  {% if sample.alt_frequency and sample.alt_frequency != -1 and sample.alt_frequency == sample.alt_frequency %}
```

Comment:
This is very weird, but I understand that is a trick so you are not displaying NaNs. But still weird!

Reply:
Ah it was still a work in progress! Never mind then

Reply:
Right, I was just testing it out! It is super weird; I was considering adding a custom filter but apparently this one is commonly used, and the Jinja devs had slightly sour comments to requests for a built in NaN filter, or modifying the current `is number` test to actually also test for NaN in case the type is correctly identified as number..
It actually seems to be working: 
![Screenshot 2025-07-04 at 14 57 30](https://github.com/user-attachments/assets/e0e944a1-c45a-49e8-8ee3-8171cfa191da)